src/displayapp/screens/WatchFaceInfineat.h

```diff
@@ -52,6 +53,7 @@ namespace Pinetime {
         Utility::DirtyValue<bool> isCharging {};
         Utility::DirtyValue<bool> bleState {};
         Utility::DirtyValue<bool> bleRadioEnabled {};
+        bool alarmState {};
```

Comment:
suggest rename to `isAlarmSet`, and maybe use `Utility::DirtyValue<bool>` (didn't check if it makes sense here, but please check if it may be useful here to prevent too many redraws)

```suggestion
        Utility::DirtyValue<bool> isAlarmSet {};
```